src/overlays/actors/ovl_Lamp_Light/ef_lamp_light.h

```diff
@@ -12,7 +13,10 @@ typedef void (*Lamp_LightActionFunc)(struct Lamp_Light*, struct Game_Play*);
 
 typedef struct Lamp_Light {
     /* 0x000 */ Actor actor;
-    /* 0x174 */ UNK_TYPE1 unk_174[0x18];
+    /* 0x174 */ s32 unk_174;
+    /* 0x178 */ Lights unk_178;
+    /* 0x174 */ char unk_186[0x2];
+    /* 0x188 */ LightNode* unk_188;
```

Comment:
`unk_186` seems to be just struct padding, so you could just remove it
```suggestion
    /* 0x178 */ Lights unk_178;
    /* 0x188 */ LightNode* unk_188;
```